radixdlt-cli/src/main/groovy/com/radixdlt/cli/ShowValidatorConfig.groovy

```diff
@@ -33,18 +34,25 @@ class ShowValidatorConfig implements Runnable {
 	@CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
 	Composite.IdentityInfo identityInfo
 
-	void run() {
+	@CommandLine.Option(names = ["-d", "--address"], paramLabel = "ADDRESS", description = "Address to which message is sent", required = false)

```

Comment:
The description for this option doesn't seem very apt now.

Reply:
Whoops, yes, thanks.